sonar-plugin-api/src/main/java/org/sonar/api/issue/Issuable.java

```diff
@@ -133,16 +139,14 @@
   boolean addIssue(Issue issue);
 
   /**
-   * Unresolved issues, including the issues reported by end-users.
-   * <p/>
-   * {@link org.sonar.api.batch.Decorator}s calling this method must be annotated with {@code @DependsUpon(DecoratorBarriers.ISSUES_TRACKED)}.
+   * @deprecated since 5.2 no more decorators on batch side
    */
+  @Deprecated
   List<Issue> issues();
 
   /**
-   * Issues marked as resolved during this scan.
-   * <p/>
-   * {@link org.sonar.api.batch.Decorator}s calling this method must be annotated with {@code @DependsUpon(DecoratorBarriers.ISSUES_TRACKED)}.
+   * @deprecated since 5.2 no more decorators on batch side
    */
+  @Deprecated
   List<Issue> resolvedIssues();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)